scripts/st2bootstrap-el8.template.sh

```diff
@@ -1,405 +0,0 @@
-          PASSWORD="${i#*=}"
-          shift
-          ;;
-          *)
-          # unknown option
-          ;;
-      esac
-    done
-
-  if [[ "$REPO_TYPE" != '' ]]; then
-      REPO_PREFIX="${REPO_TYPE}-"
-  fi
-
-  if [[ "$VERSION" != '' ]]; then
-    if [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+dev$ ]]; then
-      echo "$VERSION does not match supported formats x.y.z or x.ydev"
-      exit 1
-    fi
-
-    if [[ "$VERSION" =~ ^[0-9]+\.[0-9]+dev$ ]]; then
-     echo "You're requesting a dev version! Switching to unstable!"
-     RELEASE='unstable'
-    fi
-  fi
-
-  echo "########################################################"
-  echo "          Installing st2 $RELEASE $VERSION              "
-  echo "########################################################"
-
-  if [[ "$REPO_TYPE" == "staging" ]]; then
-    printf "\n\n"
-    echo "################################################################"
-    echo "### Installing from staging repos!!! USE AT YOUR OWN RISK!!! ###"
-    echo "################################################################"
-  fi
-
-  if [[ "$DEV_BUILD" != '' ]]; then
-    printf "\n\n"
-    echo "###############################################################################"
-    echo "### Installing from dev build artifacts!!! REALLY, ANYTHING COULD HAPPEN!!! ###"
-    echo "###############################################################################"
-  fi
-
-  if [[ "$USERNAME" = '' || "$PASSWORD" = '' ]]; then
-    echo "Let's set StackStorm admin credentials."
-    echo "You can also use \"--user\" and \"--password\" for unattended installation."
-    echo "Press \"ENTER\" to continue or \"CTRL+C\" to exit/abort"
-    read -e -p "Admin username: " -i "st2admin" USERNAME
-    read -e -s -p "Password: " PASSWORD
-
-    if [[ "${PASSWORD}" = '' ]]; then
-        echo "Password cannot be empty."
-        exit 1
-    fi
-  fi
-}
-
-
-# include:includes/common.sh
-# include:includes/rhel.sh
-
-
-# Note that default SELINUX policies for RHEL8 differ with Rocky8. Rocky8 is more permissive by default
-# Note that depending on distro assembly/settings you may need more rules to change
-# Apply these changes OR disable selinux in /etc/selinux/config (manually)
-adjust_selinux_policies() {
-  if getenforce | grep -q 'Enforcing'; then
-    # SELINUX management tools, not available for some minimal installations
-    sudo yum install -y policycoreutils-python-utils
-
-    # Allow rabbitmq to use '25672' port, otherwise it will fail to start
-    sudo semanage port --list | grep -q 25672 || sudo semanage port -a -t amqp_port_t -p tcp 25672
-
-    # Allow network access for nginx
-    sudo setsebool -P httpd_can_network_connect 1
-  fi
-}
-
-install_net_tools() {
-  # Install netstat
-  sudo yum install -y net-tools
-}
-
-install_st2_dependencies() {
-  # RabbitMQ on RHEL8 requires module(perl:5.26
-  if is_rhel; then
-    sudo yum -y module enable perl:5.26
-  fi
```

Comment:
This section is also not in the new scripts. Did you drop everything that used `is_rhel` on purpose?

Reply:
Agreed. Let's keep the firewall block you just added (thanks!) and drop this until someone complains and shows why it's required.